PurchasesTests/Mocks/MockBackend.swift

```diff
@@ -0,0 +1,209 @@
+//
+// Created by RevenueCat on 2/28/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+class MockBackend: RCBackend {
```

Comment:
there's already a `MockBackend` defined in `PurchaseTests` but it seemed a bit big and complex to reuse. I figured the easiest path would be to create a new one and use it in places other than `PurchaseTests`